oauthenticator/google.py

```diff
@@ -14,6 +14,7 @@
 
 class GoogleOAuthenticator(OAuthenticator, GoogleOAuth2Mixin):
     user_auth_state_key = "google_user"
+    _credentials = None
```

Comment:
Can you rename this, and the associated methods, to something like `service_credentials` to make it clear these aren't related to user credentials?

Reply:
This will make things much clearer, thanks @manics.